tekton-catalog/pipeline-loops/README.md

```diff
@@ -0,0 +1,32 @@
+# Custom Task: Pipeline Loops
+
+`Pipeline Loops` is implements based on [Tekton Custom Task](https://github.com/tektoncd/community/blob/master/teps/0002-custom-tasks.md) and the template of [Knative sample controller](https://github.com/knative-sandbox/sample-controller)
+
+# Goal
+`Pipeline Loops` is trying to provide a pipeline level loops to handle withItems loop in `Pipeline`(Tekton).
+
+# Take a try
+- Generate the code of client
+  `go mod vendor; go mod tidy; bash hack/update-codegen.sh`
```

Comment:
I have to upgrade my local machine go version due to some other project. On go 1.15.3 I'm see this error on my go mod vendor.
```
go: finding module for package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/informers/pipelineloop/v1alpha1/pipelineloop
go: finding module for package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/clientset/versioned
go: finding module for package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/client
go: finding module for package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/client/fake
go: finding module for package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/informers/pipelineloop/v1alpha1/pipelineloop/fake
go: finding module for package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/listers/pipelineloop/v1alpha1
github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun imports
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/clientset/versioned: module github.com/kubeflow/kfp-tekton@latest found (v0.5.0), but does not contain package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/clientset/versioned
github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun imports
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/client: module github.com/kubeflow/kfp-tekton@latest found (v0.5.0), but does not contain package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/client
github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun imports
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/informers/pipelineloop/v1alpha1/pipelineloop: module github.com/kubeflow/kfp-tekton@latest found (v0.5.0), but does not contain package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/informers/pipelineloop/v1alpha1/pipelineloop
github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun imports
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/listers/pipelineloop/v1alpha1: module github.com/kubeflow/kfp-tekton@latest found (v0.5.0), but does not contain package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/listers/pipelineloop/v1alpha1
github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun tested by
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun.test imports
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/client/fake: module github.com/kubeflow/kfp-tekton@latest found (v0.5.0), but does not contain package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/client/fake
github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun tested by
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun.test imports
	github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/informers/pipelineloop/v1alpha1/pipelineloop/fake: module github.com/kubeflow/kfp-tekton@latest found (v0.5.0), but does not contain package github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops/pkg/client/injection/informers/pipelineloop/v1alpha1/pipelineloop/fake
```

it's probably just my env. can you also populate the pkg/client as part of this PR? Thanks

Reply:
I commit the code of client as well, to avoid user generate the client code again.